app/components/item/description.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Row, Col, UncontrolledTooltip } from 'reactstrap';
+
+import Rating from '../card/Rating';
+
+type Props = {
+  title: string,
+  runtime: Object,
+  genres: Array,
+  summary: string,
+  rating: number,
+  onTrailerClick: Function,
+  year: number,
+  torrentHealth: string,
+  certification: string,
+  nrSeeders: number,
+  trailer: string
+};
+
+export default function Description({
+  title,
+  runtime,
+  genres,
+  summary,
+  rating = 'n/a',
+  onTrailerClick,
+  year,
+  torrentHealth,
+  certification,
+  nrSeeders = 0,
+  trailer
+}: Props) {
+  const torrentHealthClassName = classNames([
+    'torrent__health',
+    { [`torrent__health--${torrentHealth}`]: torrentHealth }
+  ]);
+
+  return (
+    <Col sm="6" className="Movie">
+      <h1 className="row-margin" id="title">
+        {title}
+      </h1>
+      <Row>
+        {(runtime.hours || runtime.minutes) && (
+          <span className="col-sm-3" id="runtime">
+            <h6>
+              {runtime.hours ? `${runtime.hours} hrs ` : ''}
+              {runtime.minutes ? `${runtime.minutes} min` : ''}
+            </h6>
+          </span>
+        )}
+        <span className="col-sm-9" id="genres">
+          {genres && <h6>{genres.join(', ')}</h6>}
+        </span>
+      </Row>
+      <h6 className="row-margin item__summary">{summary}</h6>
```

Comment:
Sorry I didn't mention this earlier. In tests, query selecting based on class names is pretty brittle because if they change then the tests need to change. The method I've been using is to use `data-e2e` attributes on DOM components and then querying by `[data-e2e="my-component"]` query selector in e2e tests. I think we should follow this convention here as well.